deployments/installer/install.sh

```diff
@@ -26,10 +27,14 @@ parse_args_and_install() {
       --test)
         stage="test"
         ;;
-      --ingest)
+      --ingest-url)
```

Comment:
Should this be noted as a breaking change?

Reply:
No, because nobody should be using the installer script in an automated way.  That is an abuse of the installer script.

I updated the usage output of the installer.sh script so that will tell people what the flag is when they go to use the script after it is updated.